### ISSUE TITLE
UI suggestion - maybe a less jiggly view?

### ISSUE THREAD

**Issue by danbri** [tomayac/wikipedia-live-monitor#1]
I just tried leaving this open on my 2nd monitor, alongside https://www.google.com/trends/hottrends/visualize - but found it too "jiggly". The updates are too fast to read and many of the clusters lack human readable labels, which makes it hard to get a useful topical overview. Maybe there is scope for a different kind of view, perhaps more gently scrolling only human-comprensible labels so that the window can be left open without being too distracting? It would be great to have something you could glance at and get a sense for what's happening in the wikis right now...

**Comment by tomayac** [comment]
I agree, @danbri, he UI needs some love and is very busy. Hope to find time to play around with expandable sections. For the concrete use case, consider my [Wikipedia screensaver](http://tomayac.github.io/wikipedia-screensaver/).
